README.md

```diff
@@ -33,17 +33,4 @@ These curves can be reproducibly obtained
 
 ## License
 
-Licensed under either of
-
- * Apache License, Version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or
-   http://www.apache.org/licenses/LICENSE-2.0)
- * MIT license ([LICENSE-MIT](LICENSE-MIT) or http://opensource.org/licenses/MIT)
-
-at your option.
-
-### Contribution
-
-Unless you explicitly state otherwise, any contribution intentionally
-submitted for inclusion in the work by you, as defined in the Apache-2.0
-license, shall be dual licensed as above, without any additional terms or
-conditions.
+This repository is [MIT/Apache 2.0 dual-licensed](COPYING.md).
```

Comment:
The existing preamble above is an ecosystem-wide standard for Rust crates, because what crates.io shows is the `README.md`, and the `Cargo.toml`'s `license` field, not any `COPYING.md` file. I would greatly prefer that the above preamble is left exactly as-is, and we instead duplicate it in `COPYING.md` for GitHub's benefit. If you particularly want a link from the `README.md` to `COPYING.md`, linkify "Licensed under" in the first sentence.